src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,112 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int capacity = DEFAULT_CAPACITY;
+    private Node<K, V> []table = new Node[capacity];
+    private int size = 0;
+    private int threhold = 0;
 
     @Override
     public void put(K key, V value) {
+        Node<K, V> node = new Node<>(key, value);
```

Comment:
The creation of a new node should be done after the check for resizing. If the array needs to be resized, the hash of the node may change.